### PR TITLE
Add code execution

### PR DESCRIPTION
**Pull request by alex-setyawan**
Added:
- code execution logic
- default templates
- reset button

### CODE REVIEW

**Review by seewhyjay — COMMENTED** [review]
Seems like this can be done within the frontend without the need for a microservice

File: ExecutionService/app.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    const response = await axios.post(apiEndpoint, {
+      "language": String(language),
+      "version": versionMap[String(language)],
+      "files": [
+          {
+              "content": code
+          }
+      ],
+    });
+
```
Comment (seewhyjay) [inline ExecutionService/app.js:33]: Just call this endpoint in the frontend, and do the same error handling there

**Review by seewhyjay — CHANGES_REQUESTED** [review]
Just call this endpoint in the frontend, and do the same error handling there

**Review by lambraydon — APPROVED** [review]
LGTM

**Review by seewhyjay — APPROVED** [review]

**Pull request merged**